Libraries/LibJS/Interpreter.h

```diff
@@ -136,6 +136,13 @@ class Interpreter : public Weakable<Interpreter> {
 
     bool in_strict_mode() const { return m_scope_stack.last().scope_node->in_strict_mode(); }
 
+    Vector<Value> arguments() const
```

Comment:
It's sad to return a `Vector` by value here since that means we have to copy it (meaning heap allocation.)
Let's use an iteration helper to avoid it:

```cpp
template<typename Callback>
void for_each_argument(Callback callback)
{
    if (m_call_stack.is_empty())
        return;
    for (auto& value : m_call_stack.last().arguments)
        callback(value);
}
```

Then you can use that wherever you want to iterate over the current arguments:

```cpp
interpreter.for_each_argument([&](auto& argument) {
    do_stuff_with(argument);
});
```